hnn_core/tests/test_dipole.py

```diff
@@ -48,6 +48,23 @@ def test_dipole(tmpdir, run_hnn_core_fixture):
                        "average of 2 trials"):
         dipole_avg = average_dipoles([dipole_avg, dipole_read])
 
+    # average an n_of_1 dipole list
+    single_dpl_avg = average_dipoles([dipole])
+    for dpl_key in single_dpl_avg.data.keys():
+        np.allclose(dipole_read.data[dpl_key],
+                    single_dpl_avg.data[dpl_key], rtol=0, atol=0.000051)
+
+    # average dipole list with one dipole object and a zero dipole object
+    dpl_null = dipole_read
+    dpl_null.data['agg'] = np.zeros(len(dpl_null.data['agg']))
+    dpl_null.data['L2'] = np.zeros(len(dpl_null.data['L2']))
+    dpl_null.data['L5'] = np.zeros(len(dpl_null.data['L5']))
+    dpl_1 = [dipole, dpl_null]
+    dpl_avg = average_dipoles(dpl_1)
+    for dpl_key in dpl_avg.data.keys():
+        np.allclose(dpl_1[0].data[dpl_key],
+                    dpl_avg.data[dpl_key], rtol=0, atol=0.000051)
```

Comment:
```suggestion
                    dpl_avg.data[dpl_key])
```

very strange these numbers are. I think we should just go with the defaults if it works

Reply:
@jasmainak You're right, it works without the defaults. I set the numbers because I saw it used in the previous tests and figured that those were the accepted tolerances.